client/task-list/index.js

```diff
@@ -32,6 +32,7 @@ import {
  * Internal dependencies
  */
 import './style.scss';
+import '../homescreen/style.scss';
```

Comment:
Confirming that this fixes the issue, however I really think we should be separating out those styles specific to the homescreen and those used for the dashboard.

This probably means that `woocommerce-homescreen-card` should be renamed to something like `woocommerce-dashboard-card` and separated into a dashboard stylesheet.